src/internal/api/packages/find.go

```diff
@@ -44,11 +44,38 @@ func findPackage(pattern *regexp.Regexp, w http.ResponseWriter, setDir func() (s
 		return
 	}
 
-	files, err := utils.RecursiveFileList(targetDir, pattern)
+	files, err := recursiveFileListSkipPermissionErrors(targetDir, pattern)
```

Comment:
Thats not the only thing thats changed for this, it is also using the entire file path against the regex instead of the title against the file. Should I bring that into the utils one as well.

Reply:
I added the required flags and updated existing references to maintain their current behavior.